common/footer.js

```diff
@@ -8,14 +8,16 @@
     'use strict';
 
     angular.module('chaise.footer', [])
-        .directive('footer', ['ERMrest', '$timeout', function(ERMrest, $timeout) {
+        .directive('footer', ['ERMrest', '$timeout', 'ConfigUtils', function(ERMrest, $timeout, ConfigUtils) {
+
+          var chaiseConfig = ConfigUtils.getConfigJSON();
 
             return {
                 restrict: 'E',
                 scope: {},
                 templateUrl: '../common/templates/footer.html',
                 link: function(scope, ele) {
-                    var footerText = chaiseConfig.footerMarkdown;
+                    var footerText = chaiseConfig.footerMarkdown ? chaiseConfig.footerMarkdown : "**Please check** [Privacy Policy](/privacy-policy/){target='_blank'}";
```

Comment:
This should be `var footerText = $rootScope.chaiseConfig.footerMarkdown`. The property should be defined because of `setConfigJSON()`.